src/uct/ib/base/ib_iface.c

```diff
@@ -157,7 +157,20 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    "IB Hop limit / RoCEv2 Time to Live. Should be between 0 and 255.\n",
    ucs_offsetof(uct_ib_iface_config_t, hop_limit), UCS_CONFIG_TYPE_UINT},
 
-  {"LID_PATH_BITS", "0-17",
+  {"NUM_PATHS", "auto",
+   "Number of paths to expose for the interface. 'auto' means use "
```

Comment:
maybe explain what interface is in this case? does the user know what it means here?